src/renderers/shared/shared/event/EventPluginHub.js

```diff
@@ -127,6 +127,10 @@ var EventPluginHub = {
     // live here; needs to be moved to a better place soon
     if (typeof inst.tag === 'number') {
       const props = inst.memoizedProps;
+      if (!props) {
+        // Work in progress.
```

Comment:
This is part of TODO 2 because if we have no event listeners, then we don't update the fiber pointer. If we don't update the Fiber pointer, then the new one becomes work in progress. If we try to read from work in progress, this might be null. If it completes, it might have event listeners on it that are not yet committed.

Therefore this isn't a great solution.